pennylane/templates/embeddings/iqp.py

```diff
@@ -15,19 +15,18 @@
 Contains the ``IQPEmbedding`` template.
 """
 # pylint: disable-msg=too-many-branches,too-many-arguments,protected-access
-from collections import Sequence
+from collections import Sequence, Iterable
 from itertools import combinations
-
 from pennylane.templates.decorator import template
 from pennylane.ops import RZ, MultiRZ, Hadamard
 from pennylane.templates import broadcast
 from pennylane.templates.utils import (
     check_shape,
-    check_wires,
     check_type,
     get_shape,
     check_no_variable,
 )
+from pennylane.wires import Wires, WireError
```

Comment:
Do we need `WireError`? I've seen it imported a few times but not sure if I've seen it used :thinking:

Reply:
You are totally right, not used. Pycharm always collapses the import statements, so I missed that one. Usually code quality checks pick it up though...

Reply:
```suggestion
from pennylane.wires import Wires
```